snapcraft.yaml

```diff
@@ -820,6 +820,7 @@ parts:
       - --disable-vdi
       - --disable-vnc
       - --disable-xen
+      - --disable-xkbcommon
```

Comment:
What does this do, considerations of impact?

Reply:
The markdown formatting eat the relevant bit of the commit, I've put them back now.